### ISSUE TITLE
Check YtrayClock

### ISSUE THREAD

**Issue by olluz** [White-Tiger/T-Clock#260]
I've discovered this recently:
[YTrayClock](http://ygp.orgfree.com/ytrayclock/index.html)
Works like a charm...
Maybe an option if you are having issues with T-Clock.

**Comment by blackcrack** [comment]
is there a Gitlab or an Github link ?

would be nice if anyone could maybe but still developing further T-Clock ...

**Comment by olluz** [comment]
I'm not aware of a Github. Maybe try contacting the author of the software.
T-Clock hasn't had any updates in a long time...

**Comment by blackcrack** [comment]
maybe your english is a way better than my, could you ?

**Comment by youxiao** [comment]
is it works for windows 11 ?

**Comment by olluz** [comment]
@blackcrack no, for me it's really no issue whether it is hosted on Github or not. I don't see a necessity. If you are looking for a replacement for T-Clock that works than go ahead and give it a try.
@youxiao I am not using Windows 11, but I guess it is is using the same injection mechanism that T-Clock is using to change the tray clock. So, if that has been removed from Windows 11 than it probably won't.

**Comment by blackcrack** [comment]
well, if it's works in 10, should it works also in 11, at the homepage it's 10 testet, so should it also works..,
if you whant have the classic, go to Openshell issues and search the issue because Win7 and Win10 Tasklist, there have i look for the regweaks for classic surface or at my account unter my own "Blacky"-> issues
and @olluz, i am not a programmer i be a Extended User and i had never start at c or python, i had ever only used and play around with bash and bat scripts in dos 6.22 and redhat6.0/MDK6.2 upwards :)

**Comment by GalaxySH** [comment]
> well, if it's works in 10, should it works also in 11

Provided the unanticipated deprecation of T-Clock Redux, that is evidently not the case.

**Comment by blackcrack** [comment]
@GalaxySH
like i have also read in a newsletter ( don't remember where ) the systray want also totally change and no accepting forager/stranger programs anymore how it was in the past.. the apis would be change in generally in 11 :\
because this, want T-Clock in future no more works.. Microsoft take away the totally possibility to customizing Windows like the Users want and take away the modding possibility's .. hmm.. it's time to bring up Reactos.. i feel me like a frog ... it cook's slowly..
damn.. Win11 is not "eternal" by 1/0 and should killed with Reactos to have a free, free and independence WinNT OS around the world .. .. you could maybe let to not make explode.. but you could change it ..

**Comment by Skyyblaze** [comment]
YTrayClock doesn't work on Windows 11:

![grafik](https://user-images.githubusercontent.com/22203339/126605523-10c88158-c511-4882-b487-3da3595af413.png)

**Comment by blackcrack** [comment]
Win11 is imho bullschit, it's only something for Factory's where need unit porridge for the work, nothing for private Citizen/Peoples
